recipes/crms2map/LICENSE

```diff
@@ -0,0 +1,21 @@
+MIT License
```

Comment:
The license file is already in the source distribution so you don't need to include it again here.

Reply:
Delete the file.